14.serial_search/serial_search.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'date'
+
+def serial_search(input)
+  year = 1900 + (input[0] + input[4]).to_i
+  days = input[1..3].to_i
+  production = input[5..7].to_i
```

Comment:
```suggestion
  production_no = input[5..7].to_i
```

番号だよ、数字だよ、というのがわかる名前の方がいいかも